### PR TITLE
correct links for the new fundamental pages in DevDocs

### PR DESCRIPTION
**Pull request by angelampcosta**

### CODE REVIEW

**Review by marciocloudflare — APPROVED** [review]

File: products/bots/src/content/_partials/_bots-cookie.md [hunk 1]
```diff
@@ -1 +1 @@
-Cloudflare uses the `__cf_bm cookie` to identify bots. For more details, see [Understanding Cloudflare Cookies](https://support.cloudflare.com/hc/articles/200170156#12345681).
\ No newline at end of file
+Cloudflare uses the `__cf_bm cookie` to identify bots. For more details, see [Understanding Cloudflare Cookies](https://developers.cloudflare.com/fundamentals/get-started/cloudflare-cookies).
```
Comment (pedrosousa) [inline products/bots/src/content/_partials/_bots-cookie.md]: You should also update the page title if you changed it (Now it's just "Cloudflare Cookies").
Also, update "see" to "refer to".

File: products/firewall/src/content/recipes/require-specific-http-ports.md [hunk 2]
```diff
@@ -4,7 +4,7 @@ pcx-content-type: configuration
 
 # Require specific HTTP ports
 
-By default, Cloudflare allows requests on a number of different HTTP ports (see [_Identifying network ports compatible with Cloudflare's proxy_](https://support.cloudflare.com/hc/en-us/articles/200169156-Identifying-network-ports-compatible-with-Cloudflare-s-proxy) on the Cloudflare support site).
+By default, Cloudflare allows requests on a number of different HTTP ports (see [_Identifying network ports compatible with Cloudflare's proxy_](https://developers.cloudflare.com/fundamentals/get-started/network-ports).
```
Comment (pedrosousa) [inline products/firewall/src/content/recipes/require-specific-http-ports.md]: Same comment as before (update page name, replace "see" with "refer to").

File: products/fundamentals/docs-config.js [hunk 3]
```diff
@@ -7,7 +7,7 @@ module.exports = {
   externalLinks: [
     {
       title: "How does Cloudflare work?",
```
Comment (pedrosousa) [inline products/fundamentals/docs-config.js]: Update page name.

File: products/bots/src/content/_partials/_bots-cookie.md (hunk 1 above)
Comment (angelampcosta) [reply products/bots/src/content/_partials/_bots-cookie.md]: Thanks, Pedro. I corrected the pages names and changer to "refer to".

**Pull request merged**